components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -75,11 +94,16 @@ func (c *Collector) RecordError() {
 }
 
 // RecordLatency records a latency metric
-func (c *Collector) RecordLatency(duration time.Duration) {
-	c.latency.WithLabelValues().Observe(duration.Seconds())
+func (c *Collector) RecordLatency(duration time.Duration, statusCode int, destSvc string) {
+	c.latency.WithLabelValues(fmt.Sprint(statusCode), destSvc).Observe(float64(duration.Milliseconds()))
 }
 
 // RecordEventType records a eventType metric
 func (c *Collector) RecordEventType(eventType, eventSource string) {
 	c.eventType.WithLabelValues(eventType, eventSource).Inc()
 }
+
+// RecordRequests records the eventRequests metric
+func (c *Collector) RecordRequests(statusCode int, destSvc string) {
```

Comment:
(nits) same, lets also change to argument to `responseCode` to be consistent.

Reply:
see above :)